Updating docs for sec issues (#338)
Aiven-Open/klaw-docs: docs/setup-configuration/klaw-installation/run-docker.md

   - **Username:** `superadmin`
   - **Password:** `kwsuperadmin123$$` (for Klaw releases 2.5.0 and older)
   - **Password:** `welcometoklaw` (starting from the Klaw 2.6.0 release)

>    - **Password:** based on configuration of KLAW_SUPERADMIN_DEFAULT_PASSWORD (starting from the Klaw 2.9.0 release)

Reviewer: 2.10.0 !

>    - **Password:** Set based on the `klaw.superadmin.default.password` property in `core/resources/application.properties`. Available starting in Klaw 2.9.0.

Reviewer: Although I think that these have always been able to be set here? so maybe set from 2.10.0 isnt exactly correct and we should just call out mandatory to set it yourself from version 2.10.0

>    - **Password:** Set based on the KLAW_SUPERADMIN_DEFAULT_PASSWORD property defined in Docker compose file. From Klaw 2.10.0 we are enforcing users to set their own superadmin password by setting KLAW_SUPERADMIN_DEFAULT_PASSWORD in the Docker yaml file

Reviewer 2: ```suggestion
   - **Password:**  Set using the `KLAW_SUPERADMIN_DEFAULT_PASSWORD` property in the Docker Compose file. Starting with Klaw 2.10.0, users must define their own superadmin password by setting this property in the Docker YAML file
```

trying to avoid using 'we'.
